README.md

```diff
@@ -23,19 +23,22 @@ _Note_: Sessions are required for this strategy.
     ```elixir
     config :ueberauth, Ueberauth,
       providers: [
-        twitter: {Ueberauth.Strategy.Twitter, []}
+        twitter: {Ueberauth.Strategy.Twitter, [authorize_url: "/oauth/authenticate"]}
```

Comment:
Does it fix `** (Ueberauth.Strategy.Twitter.OAuth.ApiError) Could not authenticate you. (Code 32)` ?